src/trunk.c

```diff
@@ -4616,6 +4616,19 @@ trunk_btree_pack_req_init(trunk_handle   *spl,
                        spl->heap_id);
 }
 
+static void
+trunk_compact_bundle_cleanup_iterators(trunk_handle           *spl,
```

Comment:
Minor point on the name of this function. It calls two other functions, below, one of which is named `destroy` and the other is named `deinit`. And this one is named `cleanup`.

Can we settle on naming this, say, just `trunk_compact_bundle_deinit_iterators()`, so that we are at least consistent with the name of one of the callees?

Reply:
I agree with you that this is slightly annoying, but there is a logic to the naming deinit vs. destroy.  init/deinit functions operate on an object that is already allocated (e.g. on the stack).  create/destroy allocate/deallocate the object in addition to initializing/deinitializing it.  I think it is good to have different names for those different styles of functions.

Unfortunately, this cleanup function does both!  So I think renaming it to deinit or destroy would both be wrong.  So I'm leaving it as cleanup for now.